doc/BUILD

```diff
@@ -156,4 +156,12 @@ py_test_run_all_subdirectory(
     exclude = [],
     extra_srcs = [],
     tags = ["exclusive", "team:ml"],
-)
\ No newline at end of file
+)
+
+py_test(
```

Comment:
ah i sorry i should clarified a bit more about the BUILD file, we're still migrating this part for our docs and the serve snippet tests lives in serve's BUILD file instead: https://sourcegraph.com/github.com/ray-project/ray/-/blob/python/ray/serve/BUILD?L373 

We will need to move serve docs to ray level `doc` folder again sometime when it stabilizes but not a burden on your end.

Can you also try running `bazel test //python/ray/serve:snippet_model_ensemble` after it's moved to ensure it's executable ?